.github/workflows/developer_onboarding_notification.yml

```diff
@@ -0,0 +1,73 @@
+
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
```

Comment:
Same suggestion as above, also suggest using underscores instead of hyphens for consistency.